src/Binaron.Serializer.Benchmark/Binaron.Serializer.Benchmark.csproj

```diff
@@ -4,7 +4,15 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <RootNamespace>BinSerializerTest</RootNamespace>
-  </PropertyGroup>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+  </ItemGroup>
+

```

Comment:
The benchmark project doesn't get published as a nuget package. Wouldn't have any benefits adding SourceLink, would it?

Reply:
no harm anyway, PrivateAssets="All"

![image](https://user-images.githubusercontent.com/3457140/85835961-8dcd9e00-b79e-11ea-848c-8dc6538bbeba.png)